onmt/modules/Decoder.lua

```diff
@@ -353,6 +353,7 @@ function Decoder:forward(batch, initialStates, context)
                                          { batch.size, self.args.rnnSize })
   if self.train then
     self.inputs = {}
+    self.network:apply(function(m) if m.noiseInit then m.noiseInit[1]=0 end end)
```

Comment:
It's difficult to understand the purpose of this line in the current context. Could it be hidden within the `VDropout` module in some way?